catalog/core/catalog-core-ftp/src/main/java/ddf/catalog/core/ftp/FtpServerStarter.java

```diff
@@ -36,8 +40,16 @@
 
     private static final String DEFAULT_LISTENER = "default";
 
+    private static final String WANT_AUTH = "WANT";
```

Comment:
This should default to WANT, but it should be configurable to NEED if someone wants to require 2-way TLS.